pandas/core/resample.py

```diff
@@ -483,6 +483,7 @@ def _groupby_and_aggregate(self, how, *args, **kwargs):
         result = self._apply_loffset(result)
         return self._wrap_result(result)
 
+    # pylint: disable=access-member-before-definition
     def _apply_loffset(self, result):
```

Comment:
can you fetch and rebase onto upstream/main? this function doesn't exist anymore

Reply:
I have successfully fetched and rebased, but it seems that `access-member-before-definition`  is not listed inside `pyproject.toml` anymore.